src/ca/bcit/jfx/controllers/SaveMapController.java

```diff
@@ -278,8 +278,8 @@ public void displaySaveMapWindow() {
         connNodeInput.setPromptText(resources.getString("enter_connected_nodes"));
 
         numRegeneratorInput = new TextField();
-        numRegeneratorInput.setStyle("-fx-prompt-text-fill: derive(-fx-control-inner-background, -30%);");
-        numRegeneratorInput.setPromptText("Enter regenerators (100)");
+        numRegeneratorInput.setStyle("-fx-prompt-text-fill: derive(-fx-control-inner-background, -30%);-fx-pref-width: 200");
```

Comment:
set the -fx-pref-width to 300 as one some computers, it cuts off some prompt text.